tests/pytest/run_tests.py

```diff
@@ -1,57 +1,46 @@
 #!/usr/bin/python3
 import sys
+import os
 import argparse
 import pytest
 import logging
 
-slng_types = ['ospkg', 'balabitpkg', 'custom']
-
 
 def get_parser():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-f", "--func", action="store_const", const=True, dest="func",
-                        help="Run functional tests with pytest, pytest parameters: [-lvs]")
-    parser.add_argument("-u", "--unit", action="store_const", const=True, dest="unit",
-                        help="Run unit tests with pytest, pytest parameters: [-lvs]")
-    parser.add_argument("-t", "--type", dest="type", help="Type of tested syslog_ng available types: %s" % slng_types)
-    parser.add_argument("-i", "--install-dir", dest="install_dir",
-                        help="Custom install path for syslog_ng, under this directory should be exist: [bin/, etc/, sbin/, var/]")
-    parser.add_argument("-v", "--version", dest="version", help="Syslog-ng version: [3.8, 6.0, 7.0]")
-    parser.add_argument("-l", "--log-level", dest="log_level", help="Set minimum debug log level: [INFO, DEBUG, ERROR]")
+    parser.add_argument("-r", "--running-test", dest="running_test", default="func", help="Set running test type: [func, unit]; func: run functional tests on installed syslog-ng, unit: run self-test on lib")
+    parser.add_argument("-s", "--syslog-ng-install", dest="syslog_ng_install", default="path", help="Set running syslog-ng type: [path, ospkg, balabitpkg, custom]")
+    parser.add_argument("-c", "--config-version", dest="config_version", default="3.8", help="Set config version: [3.8]")
+    parser.add_argument("-i", "--install-dir", dest="install_dir", help="Custom install path for syslog-ng, under this directory should be exist: [bin/, etc/, sbin/, var/]")
+    parser.add_argument("-l", "--log-level", dest="log_level", default="ERROR", help="Set minimum debug log level: [INFO, ERROR]")
     return parser
 
 
 def save_testdb_configuration(options):
-    configuration_file_path = "testdb_configuration.ini"
+    configuration_file_path = os.path.join("/tmp/", "testdb_configuration.ini")
```

Comment:
We should generate a new name/folder so multiple `pytest`s can run in parallel. @mitzkia says that it is difficult to inject into pytest.main....